config.go

```diff
@@ -1380,13 +1381,61 @@ func extractBitcoindRPCParams(bitcoindConfigPath string) (string, string, string
 }
 
 // checkZMQOptions ensures that the provided addresses to use as the hosts for
-// ZMQ rawblock and rawtx notifications are different.
+// ZMQ rawblock and rawtx notifications are valid and different.
 func checkZMQOptions(zmqBlockHost, zmqTxHost string) error {
-	if zmqBlockHost == zmqTxHost {
-		return errors.New("zmqpubrawblock and zmqpubrawtx must be set" +
+	// ZMQ can be used with different transports
+	// (http://api.zeromq.org/4-0:zmq-bind). At first, parse the socket URL
+	// and perform generalized comparison.
+	zmqBlockURL, err := url.Parse(zmqBlockHost)
```

Comment:
Realized that by making this change, users will now be forced to prepend their ZMQ values with `tcp://`, so it'll break some existing configs out in the wild. Because of that, I think we should defer this until we make the breaking changes to fully support bitcoind's cookie authentication.

Reply:
Hmm, I didn't think of this before. Added checks to ensure compatibility with old setups, will it go through?